RadarSDKTests/RadarAPIHelperMock.h

```diff
@@ -1,22 +0,0 @@
-//
```

Comment:
Deprecate this RadarAPIHelperMock 

Use OCMock instead https://ocmock.org/

several benefits essential to beacon trackOnce test
1. provide different mock behaviors w.r.t. passed in arguments
2. could perform verification on the arguments